README.md

```diff
@@ -32,8 +32,13 @@ address. The path is discarded.
 Options:
 
 - `timeout`, the timeout after which a request will time out, in
-  milliseconds. Default:
-  `30000` milliseconds.
+  milliseconds.
+  Default: `30e3` milliseconds (30s).
+
+- `maxAbortedRead`, Maximum number of bytes read after which an
+  aborted response will close the connection. Closing the connection
+  will error other inflight requests in pipeline.
+  Default: `1e9` bytes (1MiB).
```

Comment:
I'm not very fond of this name for the option. @szmarczak thoughts?

Reply:
It sounds good to me. Alternatively `maxAbortBytes`, `maxAbortPayload`.